src/CloudKit/Enums.cs

```diff
@@ -128,10 +135,14 @@ public enum CKReferenceAction : nuint {
 	[Native]
 	public enum CKSubscriptionType : nint {
 		Query = 1,
-		RecordZone = 2
+		RecordZone = 2,
+		[iOS (10,10), Mac (10,12)] Database = 3
 	}
 
 	// NSInteger -> CKSubscription.h
+	
+	[Introduced (PlatformName.iOS, 8, 0, message: "Use CKQuerySubscriptionOptions instead")]
```

Comment:
The existing `[iOS (8,0)]` already cover this - and this message is only useful for the deprecation.